ckanext/multilang/helpers.py

```diff
@@ -24,14 +25,17 @@ def getLanguage():
             lang = unicode(lang)
     return lang
 
+def enable_tag_localization():
+    return eval(config.get('enable_tag_localization', 'False'))
```

Comment:
Settings defined/used within an extension are usually prefixed with the extension name, for instance `multilang.enable_tag_localization`